src/load/azext_load/data_plane/load_test_run/custom.py

```diff
@@ -136,12 +137,14 @@ def stop_test_run(cmd, load_test_resource, test_run_id, resource_group_name=None
 def copy_test_run_artifacts_url(cmd, load_test_resource, test_run_id, resource_group_name=None):
     client = get_testrun_data_plane_client(cmd, load_test_resource, resource_group_name)
     logger.info("Fetching test run copy artifacts SAS URL for test run %s", test_run_id)
+    logger.warning("You can use the SAS URL in the Azure Storage Explorer or AzCopy "
```

Comment:
@saloniagrawal1997 to review

Reply:
You can use the SAS URL with Azure Storage Explorer or AzCopy to access the storage resource.